### PR TITLE
Tasks

### PR DESCRIPTION
**Pull request by Anaghakh**
Added support to perform CRUD operations on:
AWS Route53 Task Group
vDiscovery Tasks

### CODE REVIEW

File: awsrte53taskgroup.go [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+	awsRte53 := NewAwsUser(aws)
+	var err error
+	err = objMgr.connector.GetObject(awsRte53, "", &res)
+	if err != nil || res == nil || len(res) == 0 {
+		return nil, err
+	}
+
+	return &res[0], err
+}
+/*type AwsRte53Task struct {
```
Comment (chinmayb) [inline awsrte53taskgroup.go:41]: Why is this commented ?

File: awsrte53taskgroup.go [hunk 2]
```diff
@@ -0,0 +1,151 @@
+package ibclient
+
+type AwsRte53TaskOperations interface {
```
Comment (chinmayb) [inline awsrte53taskgroup.go:3]:
```suggestion
type AWSRoute53TaskOperations interface {
```

File: awsrte53taskgroup.go [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+	CreateAwsRte53TaskGroup(aws AwsRte53TaskGroup) (*AwsRte53TaskGroup, error)
+	GetAwsRte53TaskGroup(aws AwsRte53TaskGroup) (*[]AwsRte53TaskGroup, error)
+	DeleteAwsRte53TaskGroup(aws AwsRte53TaskGroup) (string, error)
+	UpdateAwsRte53TaskGroup(aws AwsRte53TaskGroup) (*AwsRte53TaskGroup, error)
+}
+
+type AwsUser struct {
```
Comment (chinmayb) [inline awsrte53taskgroup.go:10]:
```suggestion
type AWSUser struct {
```

File: awsrte53taskgroup.go (hunk 1 above)
Comment (Anaghakh) [reply awsrte53taskgroup.go:41]: The structure is not used. I'll remove that.

File: awsrte53taskgroup.go [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+	var err error
+	err = objMgr.connector.GetObject(awsRte53, "", &res)
+	if err != nil || res == nil || len(res) == 0 {
+		return nil, err
+	}
+
+	return &res[0], err
+}
+
+type AWSRte53TaskGroup struct {
```
Comment (chinmayb) [inline awsrte53taskgroup.go:42]:
```suggestion
type AWSRoute53TaskGroup struct {
```
Shall we keep it as Route it self rather than `Rte`?

Comment (Anaghakh) [reply awsrte53taskgroup.go:42]: Okay, will change that as well.

**Review by chinmayb — APPROVED** [review]

**Comment by chinmayb** [comment]
@Anaghakh please raise a PR to merge the /v2.0.0 changes to `master`

**Comment by Anaghakh** [comment]
> @Anaghakh please raise a PR to merge the /v2.0.0 changes to `master`

@chinmayb, As mentioned in the following Pull Request- https://github.com/infobloxopen/infoblox-go-client/pull/97
Few changes are made in objects.go and object_manager.go. So, it doesn't provide backward compatibility.

**Pull request merged**